ols/constants.py

```diff
@@ -232,6 +232,8 @@ class GenericLLMParameters:
 STDIO_TRANSPORT_DEFAULT_ENCODING = "utf-8"
 STDIO_TRANSPORT_DEFAULT_ENV: dict[str, str | int] = {}
 STDIO_TRANSPORT_DEFAULT_CWD = "."
+STREAMABLE_HTTP_TRANSPORT_DEFAULT_TIMEOUT = 5
+STREAMABLE_HTTP_TRANSPORT_DEFAULT_READ_TIMEOUT = 10
```

Comment:
@onmete @asamal4 Next time , lets have a comment stating that the timeouts are in `seconds`